css/index.css

```diff
@@ -1,188 +1,561 @@
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+body {
+	line-height: 1;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+ol,
+ul {
+	list-style: none;
+}
+
+blockquote,
+q {
+	quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+	content: "";
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
 }
 
 /* Your code starts here! */
 
-.container {
-	width: 880px;
-	/* border: 10px dashed black; */
-	margin: auto;
+* {
+	box-sizing: border-box;
+	padding: 10;
+	margin: 0;
+	border: 1px solid grey;
 }
 
-header {
-	/* border: 5px dashed orange; */
-	padding: 20px 20px 20px;
+header nav {
+	display: flex;
+	justify-content: space-between;
+	color: lightgray;
+	margin: 35px 60px;
+	text-align: center;
+	align-items: center;
+	width: 100%;
+}
 
+.nav-links {
+	width: 50%;
+	display: flex;
+	justify-content: space-between;
+	padding: 0 20px;
 }
 
-nav {
-	/* border: 3px solid red; */
-	display: inline-block;
-	width: 75%;
+.home {
+	display: flex;
+	width: 100%;
+	align-items: center;
+}
+
+
+
+.logoContainer {
+	display: flex;
+
 }
+
 nav a {
-	color: grey;
+	display: flex;
+	justify-content: space-between;
 	text-decoration: none;
-	padding: 0 25px 0px;
-	/* border: 3px solid blue; */
+	color: grey;
+	font-size: 13px;
+	letter-spacing: 1.5px;
+
 }
 
-.intro-container {
-	text-align: center;
-	/* border: 3px solid green; */
+nav a:hover {
+	/*styles are applied when a user hovers over this element*/
+	color: #4682b4;
+	text-decoration: overline;
 }
-.intro  {
-	/* border: 4px dashed red; */
-	display: inline-block;
-	width: 450px;
-	text-align: center;
-	padding: 0px 0px 40px;
+
+.top-page {
+	display: flex;
+	flex-grow: 2;
+	margin-bottom: 3%;
+	padding: 5px;
+	justify-content: space-around;
 }
 
- .intro h1 {
-	font-size: 50pt;
-	/* border: 1px dashed red; */
-}  
+.box {
+	display: flex;
+	flex-direction: column;
+	width: 30%;
+	justify-content: center;
+	align-items: center;
+}
 
+.main-heading {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+}
 
-.introimg {
-	width: 30%;
-	/* border: 1px dashed red; */
-	display: inline-block;
-	/* TRBL*/
-	padding: 20px 10px 10px 20px;
+button {
+	background-color: white !important;
+	color: black;
+	width: 35%;
+	height: 10%;
+	padding: 3px;
+	font-size: 10px;
+	border-radius: 12px;
+	margin-top: 2%;
 }
 
```

Comment:
you are targeting the get started button twice once with the button element selector and again with the .first class selector and best practices would be to target it with just the class selector and style the button with that.  also setting a width and height on the button is what is causing issues with it in tablet and mobile view.  padding should be used to adjust the height and width of buttons and if em units are used the button will scale properly at smaller view widths.